xpdtools/tools.py

```diff
@@ -580,3 +580,11 @@ def inner(x, *args, **kwargs):
         return func(*args, **kwargs)
 
     return inner
+
+
+def decomp(data, variance=.9):
```

Comment:
This is just returning eigenvalues, which I guess go in as QOIs, which is good.  If the user wants to plot the eigenvectors, do we want to give them that opportunity?

Reply:
Sure, I'm not certain how to get that out of the model though.